### PR TITLE
ebtables: versionMatcher remove v before version

### PR DESCRIPTION
**Pull request by defo89**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
Do not expect that `ebtables --version` has v string before the version.

**Which issue(s) this PR fixes**:

Fixes https://github.com/kubernetes/utils/issues/236

### CODE REVIEW

**Comment by aojea** [comment]
yeah, I can confirm this is the case in 

Centos8
```
ebtables --version
ebtables 1.8.4 (nf_tables)
```

Fedora
```
ebtables --version
ebtables 1.8.7 (nf_tables)
```
maybe that was changed with the migration to nftables

**Comment by aojea** [comment]
can we cover this with some unit tests to check that it works for both formats?

File: net/ebtables/ebtables.go [hunk 1]
```diff
@@ -121,7 +121,7 @@ func getEbtablesVersionString(exec utilexec.Interface) (string, error) {
 	if err != nil {
 		return "", err
 	}
-	versionMatcher := regexp.MustCompile(`v([0-9]+\.[0-9]+\.[0-9]+)`)
+	versionMatcher := regexp.MustCompile(`([0-9]+\.[0-9]+\.[0-9]+)`)
```
Comment (thockin) [inline net/ebtables/ebtables.go:124]: Maybe clearer with `v?` at the beginning?  And a comment explaining this is actually seen in the wild in both forms.

Agree with the point about a unit test.  Maybe factor this to a `parseVersion(string) (string, error)` function and test that with a few interesting inputs.

Comment (thockin) [reply net/ebtables/ebtables.go:124]: https://go.dev/play/p/C5_zp5HI8uX

**Comment by thockin** [comment]
Hi @defo89 is this still something you are working on?

**Comment by thockin** [comment]
Last ping before code freeze...

**Comment by thockin** [comment]
Labelled this good-first-issue in case someone wants to take it over.

**Pull request closed without merging**